src/main/java/com/playmonumenta/networkchat/channel/Channel.java

```diff
@@ -26,15 +26,18 @@ public abstract class Channel {
 	protected final UUID mId;
 	protected Instant mLastUpdate;
 	protected String mName;
+	protected String mDescription;
+	protected static String mUnsetDescription = "<gray>Description unset.";
```

Comment:
I'd suggest leaving this as a blank string - it'd be odd to see "Description unset" on, say, guild channels. Related note, we can update the descriptions on guild channels automatically from the main plugin, maybe give guilds the ability to set a description of their choice from there? (And why yes I do find the "no comment" review dog message annoying, but it should probably be done.)

Reply:
This also means that none of the subclasses would need updating, except to override/disable the setDescription where appropriate (such as with ChannelLoading).